.github/workflows/CI_Pipeline.yml

```diff
@@ -25,20 +25,19 @@ jobs:
     - uses: actions/checkout@v2
     - run: docker build --target test --tag my-test-image .
     - run: docker run --env-file .env.test  my-test-image tests
-    - run: docker run -e TRELLO_KEY=${{secrets.TRELLO_KEY}}  -e TRELLO_TOKEN=${{secrets.TRELLO_TOKEN}} -e TRELLO_BOARD_ID=${{secrets.TRELLO_BOARD_ID}} my-test-image tests_e2e
-
-  deploy:
-    name: Deploy build package to Heroku
-    needs: build
-    if: ${{ github.event_name ==  'push' && github.ref == 'refs/heads/exercise-8' }}
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2 # Adds a step to checkout the repository code
+    - run: docker run -e MONGO_DB_NAME=${{secrets.MONGO_DB_NAME}} -e MONGO_CONNECT=${{secrets.MONGO_CONNECT}} -e MONGO_COLLECTION_NAME=${{secrets.MONGO_COLLECTION_NAME}} -e MONGO_DEFAULT_STATUS=${{secrets.MONGO_DEFAULT_STATUS}} my-test-image tests_e2e
```

Comment:
It's overkill to make the db name, collection name, and default status into secrets.

You could just set e.g. `-e MONGO_DB_NAME=e2e_db`